android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/EngineSession.kt

```diff
@@ -329,6 +331,20 @@ abstract class EngineSession(
          * @param throwable The throwable from the exception.
          */
         fun onCheckForFormDataException(throwable: Throwable) = Unit
+
+        /**
+         * Event to indicate that the translation or original page restore completed successfully.
+         *
+         * @param operation the change the translation engine completed
```

Comment:
```suggestion
         * @param operation The translation operation that the translation engine completed.
```
Let's use complete sentence (capitalization and period at the end) for all our @param since it's already what was used before. This comment applies elsewhere in your PR so I would encourage a self-review of your PR.

Reply:
Updated! (Except where preexisting functions had a different pattern.)